brigade-generic-gateway/cmd/brigade-generic-gateway/server.go

```diff
@@ -49,13 +49,15 @@ func newRouter(store storage.Store) *gin.Engine {
 	router := gin.New()
 	router.Use(gin.Recovery())
 
-	handler := webhook.NewGenericWebhookSimpleEvent(store)
-
-	events := router.Group("/simpleevent")
-	{
-		events.Use(gin.Logger())
-		events.POST("/:projectID/:secret", handler)
-	}
+	handlerSE := webhook.NewGenericWebhookSimpleEvent(store)
+	eventsSE := router.Group("/simpleevent")
+	eventsSE.Use(gin.Logger())
+	eventsSE.POST("/:projectID/:secret", handlerSE)
+
+	handlerCE := webhook.NewGenericWebhookCloudEvent(store)
+	eventsCE := router.Group("/cloudevent")
```

Comment:
Do we want to pluralize this, as the spec is `CloudEvents`?
If we did pluralize this one, it would probably make sense to pluralize `simpleevents` as well.

Reply:
I wasn't sure about that, to be honest. I used singular because we're just using POST and NOT GET (so, POST a - single - cloud event contrary to GET - many - cloud events). However, plural is much more common in APIs.
But, as I said, no strong opinions here... whatever you think, I'm good.

Reply:
Yeah, the more I look at this, the more I like the pluralized versions.